### DISCUSSION TITLE
Does Highlight support Heroku drains?

### DISCUSSION THREAD

**Discussion by carloslfu** [highlight/highlight#7239] [Q&A] [answered]
Here is the format: https://devcenter.heroku.com/articles/log-drains

**Comment by Vadman97** [comment, marked as answer]
Hi @carloslfu , we support TLS encrypted syslog on syslog.highlight.io:34302 (which is one of the heruko supported formats). See the docs for more details: https://www.highlight.io/docs/getting-started/backend-logging/syslog

**Reply by carloslfu** [reply]
Thanks!
